host/storage/storage.go

```diff
@@ -955,6 +980,8 @@ func (vm *VolumeManager) PruneSectors() (int, error) {
 
 // NewVolumeManager creates a new VolumeManager.
 func NewVolumeManager(vs VolumeStore, a Alerts, cm ChainManager, log *zap.Logger) (*VolumeManager, error) {
+	// Initialize cache with LRU eviction and a max capacity of 1000 items
+	cache, _ := lru.New[types.Hash256, *[rhpv2.SectorSize]byte](1000)
```

Comment:
1000 sectors is 4GiB, the default maximum should be 64 (256MiB of memory).
Once the cache is configurable users with more memory can increase the cache size.